modules/services/volnoti.nix

```
@@ -0,0 +1,25 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.volnoti;
+
+in {
+  meta.maintainers = [ maintainers.imalison ];
+
+  options = {
+    services.volnoti = { enable = mkEnableOption "Volnoti volume HUD daemon"; };
+  };
+
+  config = mkIf cfg.enable {
```

```suggestion
  config = mkIf cfg.enable {
    home.packages = [ pkgs.volnoti ];
```

To be able to use `volnoti-show` without the full path from the command line for instance.